dot-net-sdk/EppoClient.cs

```diff
@@ -40,6 +40,12 @@ public class EppoClient
     }
 
 
+    public double? GetIntegerAssignment(string subjectKey, string flagKey, SubjectAttributes? subjectAttributes = null)
```

Comment:
You got it returning a `double` did you mean an `int` or `long` type ?

Reply:
ya, sure did. thanks for catching